src/App.js

```diff
@@ -1,16 +1,21 @@
 import React, { Component } from 'react';
-import { Route } from 'react-router-dom';
+import { Route ,Switch } from 'react-router-dom';
 
 
 import Layout from './hoc/Layout/Layout';
-import Todo from './components/Todo/Todo'
+import Todo from './components/Todo/Todo';
+import ChartTodo from './components/ChartTodo/ChartTodo';
+
 
 class App extends Component {
      render() {
         return (
           <div>
             <Layout>
+              <Switch>
+              <Route path="/chart" component={ChartTodo}/>
               <Route path="/" component={Todo}/>
+              </Switch>
```

Comment:
We can create constants files with all the routes and use the path from it

Reply:
thanks for this attention it's a good idea to share a path in route and link.I will do that